src/backend/distributed/commands/sequence.c

```diff
@@ -266,7 +266,16 @@ PreprocessDropSequenceStmt(Node *node, const char *queryString,
 	{
 		RangeVar *seq = makeRangeVarFromNameList(objectNameList);
 
-		Oid seqOid = RangeVarGetRelid(seq, NoLock, stmt->missing_ok);
+		Oid seqOid = RangeVarGetRelid(seq, NoLock, true);
+
+		if (!stmt->missing_ok && !OidIsValid(seqOid))
+		{
+			/*
```

Comment:
Do we really need to check `stmt->missing_ok` here? I mean, as long as `!OidIsValid(seqOid)` we should just rely on Postgres, shouldn't we?